libs/core.lua

```diff
@@ -129,6 +129,9 @@ local function makeCore(config)
       error("Not a package: " .. path)
     end
     local author, name, version = pkg.normalize(meta)
+    if not author or not name then
```

Comment:
Change to `not (author and name)` to save an opcode.  Unless luajit already does this internally.